gnuradio-runtime/examples/volk_benchmark/volk_math.py

```diff
@@ -5,7 +5,8 @@
 from gnuradio import gr
 from gnuradio import blocks
 import argparse
-from volk_test_funcs import *
+from volk_test_funcs import (create_connection, new_table, replace_results,
```

Comment:
Thank you for tackling this! You seem to have made this file executable. Mind changing that back?

Reply:
The parens are also un-Pythonic. But that's not a major issue.